xml/System.Reflection/EventInfo.xml

```diff
@@ -160,22 +160,21 @@
 ## Remarks  
  This method attempts to add a delegate to synchronize the event on the target object.  
   
- Each time the event is raised by the target parameter, the method or methods encapsulated by the handler will be invoked.  
+ Each time the event is raised by the target parameter, the method or methods encapsulated by the handler are invoked.  
   
-   
+You might use the `AddEventHander` method when you load a type after the program is already compiled, when it is not possible to use the C# += syntax or the Visual Basic `WithEvents` and `Handles` keywords to hook up the event.   
   
 ## Examples  
- The following code example shows how to load an assembly, create an instance of a type from the assembly, create an event handler using a dynamic assembly, and hook up the dynamic event handler. All actions are performed using late binding.  
+ The following example loads an assembly, creates an instance of a type from the assembly, creates an event handler using a dynamic assembly, and hooks up the dynamic event handler. All actions are performed using late binding.  
   
- The code example loads System.dll and creates an instance of the <xref:System.Timers.Timer?displayProperty=nameWithType> class. The example uses the <xref:System.Type.GetEvent%2A?displayProperty=nameWithType> method to get the <xref:System.Timers.Timer.Elapsed> event, and the <xref:System.Reflection.EventInfo.EventHandlerType%2A> property to get the delegate type for the event.  
+ The example creates an instance of the <xref:System.Timers.Timer?displayProperty=nameWithType> class. The <xref:System.Timers.Timer> instance is stored in a variable of type <xref:System.Object>, and all code that accesses the Timer does so late-bound. The example uses the <xref:System.Type.GetEvent%2A?displayProperty=nameWithType> method to get the <xref:System.Timers.Timer.Elapsed> event, and the <xref:System.Reflection.EventInfo.EventHandlerType%2A> property to get the delegate type for the event.  
```

Comment:
The one instance of "Timer" that is capitalized, and not an xref: Should it be code-fenced so it is not localized?